python/TestHarness/testers/Tester.py

```diff
@@ -312,6 +313,10 @@ def getProcs(self, options):
         """ return number of processors to use for this tester """
         return 1
 
+    def getSlots(self, options):
+        """ return number of slots to use for this tester """
+        return int(self.getThreads(options)) * int(self.getProcs(options))
```

Comment:
Why you are you casting here? These methods should both be returning ints already.

Reply:
Just being carful, because getThreads/Procs in the base class is casted. But perhaps the derived class is dumb..?  I'll change it to satisfy this comment.